submissions/Barasii/DOM-menu/css/style.css

```diff
@@ -0,0 +1,131 @@
+* {
+    margin: 0;
+    padding: 0;  
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 1.5;
+    color: #222;
+    background-color: #000;
+	background-size: cover;
```

Comment:
keep the same indentation as above

Reply:
Thank you for your attention and cooperation.